### ISSUE TITLE
Dupe with trinkets

### ISSUE THREAD

**Issue by McReiz** [Tiviacz1337/Travelers-Backpack#1068]
Mod loader and version-
Minecraft 1.20.1
Quilt 1.20.1 (0.26.4)
travelersbackpack-fabric-1.20.1-9.1.18

- Describe the bug 
Backpack duplicates with trinkets integration on.

- Write steps to reproduce the bug

1. Turn on trinkets integration.
2. Take two of the same backpacks with different items in them.
3. get 2 backpacks
4. Place one of them in the trinket slot of the backpack full of items
5. You take the second one, click on it and move it to the trinket slot, and replace it.

and so you have 2 backpacks with the same content

Video:
https://github.com/user-attachments/assets/31c76e3b-ab70-4b9e-9b30-bdf6af0d0edc

**Comment by son0fether** [comment]
+1, with curios and also this is item DESTRUCTION.  The items in the second backpack are completely wiped.

Minecraft: 1.20.1
Forge: 47.3.11
Traveler's Backpacks: 9.1.16
Curios API: 5.11.0

Steps:
1. Get two Traveler's backpacks
2. Right click with backpacks in hand, put a diamond in backpack 1 and an emerald in backpack 2
3. Wear a backpack in a Curio (anything) slot and a backpack in the Back slot
4. Use the keybind (default 'B') to open the 'worn' backpack, and add another item, like a sword
5. Take both backpacks off
6. Check inventory in backpacks by right-clicking, they will both either be sword+diamond or sword+emerald, other is GONE

So this is both item duplication and accidental item destruction, very dangerous.

**Comment by McReiz** [comment]
> +1, with curios and also this is item DESTRUCTION. The items in the second backpack are completely wiped.
> 
> Minecraft: 1.20.1 Forge: 47.3.11 Traveler's Backpacks: 9.1.16 Curios API: 5.11.0
> 
> Steps:
> 
>     1. Get two Traveler's backpacks
> 
>     2. Right click with backpacks in hand, put a diamond in backpack 1 and an emerald in backpack 2
> 
>     3. Wear a backpack in a Curio (anything) slot and a backpack in the Back slot
> 
>     4. Use the keybind (default 'B') to open the 'worn' backpack, and add another item, like a sword
> 
>     5. Take both backpacks off
> 
>     6. Check inventory in backpacks by right-clicking, they will both either be sword+diamond or sword+emerald, other is GONE
> 
> 
> So this is both item duplication and accidental item destruction, very dangerous.

That's right, I had a user abusing the bug to get more valuable items and another who lost all his items due to this same problem.

**Comment by Tiviacz1337** [comment]
Curios is fixed in latest Traveler's Backpack alpha for 1.20.1, Fabric is also going to be fixed in alpha. Alpha versions introduce upgrades and complete rewrite of backpack inventory, bugs may occur but I'm actively working on polishing and eventually marking these versions as beta and finally release

**Comment by Tiviacz1337** [comment]
https://www.curseforge.com/minecraft/mc-mods/travelers-backpack-fabric/files/all?page=1&pageSize=20
Issue resolved in latest alpha versions, make sure these versions provide new content and significant changes
